### ISSUE TITLE
[Installer] Issue downloading frontend

### ISSUE THREAD

**Issue by SchrodingersGat** [inventree/InvenTree#7338]
Ref: https://github.com/inventree/InvenTree/issues/7246#issuecomment-2132173108

```
InvenTree database migrations completed!
Updating frontend...
Downloading frontend...
fatal: not a git repository (or any of the parent directories): .git
[ERROR] Cannot get current ref via 'git rev-parse HEAD'
```

**Comment by matmair** [comment]
Sounds like a custom/master deployment of inventree

**Comment by SchrodingersGat** [comment]
@mabroens can you confirm the version of InvenTree you are installing? Is it the default "stable" version?

**Comment by mabroens** [comment]
pressing `Auto Allocate` opens the popup 👍 
![image](https://github.com/inventree/InvenTree/assets/130482028/24716efd-3ebe-4c92-b042-7d80be34c030)

pressing the submit button shoudl to my expectation result in the allocated column:
![image](https://github.com/inventree/InvenTree/assets/130482028/620ba2d8-4295-495e-b885-8236e5f0b88e)

change but nothing happens.

pressing the auto allcoate  button  a second time
![image](https://github.com/inventree/InvenTree/assets/130482028/61e38fd3-a65e-4586-b7a6-4f1df4ceacf1)

does work:
![image](https://github.com/inventree/InvenTree/assets/130482028/9c3eb2ba-11ad-4167-9a4b-b3aa4a23ec02)

**Comment by matmair** [comment]
@mabroens was this comment meant to be here? Is the frontend working now?

**Comment by mabroens** [comment]
ah sorry no was for #7335

**Comment by matmair** [comment]
Can you please fill out the normal bug report template for this issue? If you are bare metal (git) deploying that is the cause of the issue as you are missing yarn, if that is not the case pleas open a fully filled out bug report / update this bug with the info from the template so we can work in an efficient matter

**Comment by second-string** [comment]
I also see the original error of this issue (`[ERROR] Cannot get current ref via 'git rev-parse HEAD'`) on a fresh install using the installer script on Ubuntu 20.04. Here is the snip from the full installation log:
```
# inv update| Synchronizing apps without migrations:
# inv update|   Creating tables...
# inv update|     Running deferred SQL...
# inv update| Running migrations:fatal: not a git repository (or any of the parent directories): .git
/opt/inventree/env/lib/python3.9/site-packages/allauth/exceptions.py:9: UserWarning: allauth.exceptions is deprecated, use allauth.core.exceptions
  warnings.warn("allauth.exceptions is deprecated, use allauth.core.exceptions")
/opt/inventree/env/lib/python3.9/site-packages/allauth/exceptions.py:9: UserWarning: allauth.exceptions is deprecated, use allauth.core.exceptions
  warnings.warn("allauth.exceptions is deprecated, use allauth.core.exceptions")
/opt/inventree/env/lib/python3.9/site-packages/allauth/exceptions.py:9: UserWarning: allauth.exceptions is deprecated, use allauth.core.exceptions
  warnings.warn("allauth.exceptions is deprecated, use allauth.core.exceptions")

# inv update|   No migrations to apply.
# inv update| ========================================
# inv update| InvenTree database migrations completed!
# inv update| Updating frontend...
# inv update| Downloading frontend...
# inv update| [ERROR] Cannot get current ref via git rev-parse HEAD
# inv update| Python version 3.9.5 - /opt/inventree/env/bin/python3
# inv update| Rendered all files in /opt/inventree/src/backend/InvenTree/templates/js/translated
# inv update| Collecting static files...
# inv update| Python version 3.9.5 - /opt/inventree/env/bin/python3
# inv update| Python version 3.9.5 - /opt/inventree/env/bin/python3
# inv update| Python version 3.9.5 - /opt/inventree/env/bin/python3
```

Do you want a new issue for this or is it good here?

**Comment by matmair** [comment]
Related https://github.com/inventree/InvenTree/issues/5278

**Comment by matmair** [comment]
Working on this

**Comment by matmair** [comment]
@second-string a fixed release for 0.15.4 can be found [here](https://packager.io/gh/matmair/InvenTree/builds/801/install/ubuntu-20.04) - official releases of the `master` channel might take a bit as we need to fix underlying infra.

**Comment by second-string** [comment]
@matmair  I have a working install that I did on Ubuntu 20.04 LTS (to fix the python issue). I don't think I had to manually copy the frontend for that one either, but my point is that I'm not in a position to test this anymore unfortunately.

You're welcome to close the issue since the comments will serve as the historical record for anyone else that might have the issue, I'm all set. Thanks!
